virtcontainers/pkg/oci/utils.go

```diff
@@ -642,6 +642,8 @@ func addHypervisporVirtioFsOverrides(ocispec specs.Spec, sbConfig *vc.SandboxCon
 
 	if value, ok := ocispec.Annotations[vcAnnotations.VirtioFSCache]; ok {
 		sbConfig.HypervisorConfig.VirtioFSCache = value
+	} else {
+		sbConfig.HypervisorConfig.VirtioFSCache = "none"
```

Comment:
Please define it in `pkg/katautils/config-settings.go` and set it in `GetDefaultHypervisorConfig()`.

Reply:
Thanks, I set it in the wrong place.